modules/vba_documents/app/controllers/vba_documents/internal/v1/upload_complete_controller.rb

```diff
@@ -68,16 +68,23 @@ def process_upload(upload_id)
           store = VBADocuments::ObjectStore.new
           raise Common::Exceptions::RecordNotFound, upload_id unless upload && store.bucket.object(upload.guid).exists?
 
+          log_details = { 'job' => self.class.name }.merge(upload.as_json)
+
           if upload.status.eql?('pending')
-            Rails.logger.info("VBADocuments: Processing: #{upload.inspect}")
-            upload.update(status: 'uploaded')
+            Rails.logger.info("VBADocuments: Processing: #{upload.inspect}", log_details)
+
+            upload.update!(status: 'uploaded')
+
+            Rails.logger.info("VBADocuments: #{upload.class.name.demodulize} progressed to \"uploaded\" status",
+                              log_details)
```

Comment:
This was the main log that I added in order to track evidence submissions between the "uploaded" and "received" statuses. (We already had a log for the "received" status.) The `log_details` are passed as a separate argument so that each key-value pair in that hash can be queried for separately in Datadog.